action.yml

```diff
@@ -0,0 +1,37 @@
+name: Deploy to OpenShift
+description: Deploy Dockerized application to OpenShift choosing from predefined config templates
+inputs:
+  MODE:
+    description: Type of app, either "client" or "server"
+    required: true
+  AUTH_TOKEN:
+    description: OpenShift service account authorization token
+    required: true
+  NAMESPACE:
+    description: OpenShift project namespace
+    required: true
+  APP_NAME:
+    description: OpenShift app name label
+    required: true
+  SERVER_PORT:
+    description: Port exposed by the server
+    required: false
+    default: '5000'
```

Comment:
I was using numbers in the config templates. I'm assuming string doesn't matter either?